marimo/_ai/llm/_impl.py

```diff
@@ -55,6 +55,7 @@ class openai(ChatModel):
             If not provided, the API key will be retrieved
             from the OPENAI_API_KEY environment variable or the user's config.
         base_url: The base URL to use
+        stream: Whether to stream the response. Defaults to False.
```

Comment:
Can you provide more context in this docstring on what situations one can set this to `True`? Why does it default to `False`?